src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -6,6 +6,12 @@ description: "Set up Azure Active Directory single sign-on (SSO) on Sentry."
 
 ## Installation
 
+<Alert level="warning">
+
+If you rename your organization's slug, you will need to replace the previous organization slug with the new slug name in the relevant steps where you enter the SAML configuration values.
```

Comment:
Edited the wording for style and concision and to remove what seems like some redundancy in the wording. Same edit applies to all of the notes.

```suggestion
If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.
```